docs/shared/reference/glossary.md

```diff
@@ -10,6 +10,12 @@ A [project](#project) that can run on its own. Generally uses [libraries](#libra
 
 > See: [Applications and Libraries](/concepts/more-concepts/applications-and-libraries)
 
+### Atomizer
+
+The Atomizer automatically splits tasks so that they can be executed in parallel.
```

Comment:
Atomizer is natively supported by Nx Cloud Agents, providing you detailed task information in the UI, including interpreted logs and artifacts, automatically grouped by project and workflow.